docs/build_and_load.md

```diff
@@ -1,5 +1,11 @@
 # Building and Loading
 
+!!! note
+    As of version 2.4.X, the phg utilizes a new version of anchorwave(1.2.3).
+    This changes how ASM coordinates are handled.  If you are using old MAF files,
```

Comment:
I would change to "If you are using MAF files created with a phgv2 version earlier than 2.4.0 ..."